frontend/src/libs/components/modal/modal.tsx

```diff
@@ -37,18 +38,21 @@ const Modal: React.ForwardRefRenderFunction<
         <div className={styles['modal']} ref={modalReference}>
           <div className={styles['header']}>
             <span className={styles['title']}>{title}</span>
-            <div className={styles['icon-container']}>
-              <Button
-                label="Close modal"
-                iconName="close"
-                iconColor={IconColor.BLACK}
-                iconWidth={30}
-                iconHeight={30}
-                style="rounded-transparent"
-                onClick={handleClose}
-                isLabelVisuallyHidden
-              />
-            </div>
+
+            {showCrossIcon && (
```

Comment:
```suggestion
            {isCloseIconShown && (
```
Actually we renamed the icon